docs/pages/deploy-a-cluster/high-availability.mdx

```diff
@@ -112,6 +130,19 @@ services:
 </TabItem>
 </Tabs>
 
+### Configuring the Auth Service load balancer
+
+The Auth Service load balancer must forward traffic to the Auth Service's gRPC
+port. In this guide, we are assuming that you have configured the Auth Service
+load balancer to forward traffic from port `3025` to port `3025` on an available
+Auth Service instance. 
+
+If you are running your Teleport cluster on Kubernetes, you can configure your
```

Comment:
Hmm, this confuses me a bit. Auth never listens on port 3026.

Reply:
I'm basing this on the `teleport-cluster` chart, which enables the Kubernetes Service on Auth Service pods: https://github.com/gravitational/teleport/blob/64448b1b0ed36643aee0bf471171ba677548f4aa/examples/chart/teleport-cluster/templates/auth/_config.common.tpl#L5-L8